src/task.c

```diff
@@ -319,9 +326,11 @@ static void ctx_switch(jl_ptls_t ptls, jl_task_t **pt)
             jl_swap_fiber(lastt_ctx, &t->ctx);
     }
     else {
+#ifndef JL_HAVE_BYSYNCIFY
```

Comment:
Should this be:

```
#ifdef COPY_STACKS
```

like in https://github.com/JuliaLang/julia/pull/32532/files#diff-e33939bb54b7505cfc2f5d12b60d0a9cR1044

Reply:
yes